pupil_src/shared_modules/gaze_producer/worker/map_gaze.py

```diff
@@ -0,0 +1,109 @@
+"""
+(*)~---------------------------------------------------------------------------
+Pupil - eye tracking platform
+Copyright (C) 2012-2019 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+"""
+import msgpack
+from time import time
+
+import player_methods as pm
+import tasklib
+import tasklib.background.patches as bg_patches
+from calibration_routines import gaze_mapping_plugins
+
+g_pool = None  # set by the plugin
+
+
+def create_task(gaze_mapper, calibration):
+    assert g_pool, "You forgot to set g_pool by the plugin"
+    mapping_window = pm.exact_window(g_pool.timestamps, gaze_mapper.mapping_index_range)
+    pupil_pos_in_mapping_range = g_pool.pupil_positions.by_ts_window(mapping_window)
+    serialized_pupil_pos_in_mapping_range = [
+        pp.serialized for pp in pupil_pos_in_mapping_range
+    ]
```

Comment:
Since `Serialized_Dict` is pickable, do we still have to do that? Would be nice if we could hide the whole serialization complexity as much as possible.